support/runner-script.py

```
@@ -271,6 +271,128 @@ def run_merge(args):
+
+		if dry_run:
+			print(
+				f'dry-run: would mirror {name}: {original_url} -> gs://{args.bucket}/{upload_object}'
+			)
+			continue
+
+		upload_url_api = f'https://storage.googleapis.com/upload/storage/v1/b/{urllib.parse.quote(args.bucket, safe="")}/o?uploadType=media&name={urllib.parse.quote_plus(upload_object)}'
+
+		try:
+			print(f'info: downloading {name} from {original_url}')
+			with urllib.request.urlopen(original_url) as resp:
```

Ruff (0.15.10)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)